### DISCUSSION TITLE
add favicon to essentialsx.net

### DISCUSSION THREAD

**Discussion by hirata-estate** [EssentialsX/Essentials#4939] [Ideas and Feature Suggestions] [answered]
![image](https://user-images.githubusercontent.com/61897277/173249236-bd085a1f-c832-47e7-b14d-63e033d55380.png)

**Comment by mdcfe** [comment, marked as answer]
This will be done with the upcoming website rewrite.

In the future, please use the https://github.com/EssentialsX/Website repo for website-related suggestions.
